test/Turn-test.js

```diff
@@ -11,32 +11,27 @@ describe('Turn', function() {
     expect(Turn).to.be.a('function');
   });
 
-  it('should be an instance of a Turn', function() {
+  it('should be an instance of Turn', function() {
     const turn = new Turn();
-    expect(Turn).to.be.an.instanceof(Turn);
+    expect(turn).to.be.an.instanceof(Turn);
```

Comment:
Duh moments. These all needed to be lowercase since a new instance is being held in the variable right above this line. (And a couple others lower down in this file.)